quilt/tools/main.py

```diff
@@ -140,6 +140,11 @@ def check_hash(group, hashstr):
     inspect_p.add_argument("package", type=str, help=HANDLE)
     inspect_p.set_defaults(func=command.inspect)
 
+    rm_p = subparsers.add_parser("rm")
+    rm_p.add_argument("package", type=str, help=HANDLE)
+    rm_p.add_argument("-f", "--force", action="store_true", help="Remove without prompting")
+    rm_p.set_defaults(func=command.rm)
+
```

Comment:
Given a single name, are we sure about "rm" vs "uninstall" which makes it clear that this is local.  Sorry for the late question...

quilt rm/del/delete is ambiguous whether it applies server-side, client-side or both...

also, if you intend to someday add metadata-delete-only, you might want to spec this out now, so the syntax won't be awkward...

Reply:
I think 'rm' is the clearest. It matches Docker and Linux. Uninstall feels like a close second because it matches pip.